src/content/ui-components/components/code-block-button.mdx

```diff
@@ -17,27 +17,29 @@ component:
 
 import { CodeDemo } from '@/components/CodeDemo'
 
-Toggle through block-level nodes like code blocks with the code block button.
+A fully accessible code block button for Tiptap editors. Easily toggle selected content into a `<codeBlock>` with keyboard shortcut support and flexible customization options.
```

Comment:
Does the `<codeBlock>` HTML element exist? Could you review that this is accurate?